examples/gps/gps_example.cc

```diff
@@ -0,0 +1,202 @@
+//
```

Comment:
I don't think we need this example in bob_robotics

Reply:
I personally quite like it. It's quite handy to have some code to check that your GPS unit is working and has found the satellites etc.

It could maybe go in ``tools/`` rather than ``examples/``.

Reply:
If it used OpenCV or ImGUI sure, but my issue is with all the GTK dependencies it adds and the general grossness of GTK code....

Reply:
I think it's ok to have GTK as a dependency -- all Linux users will probably have it already.

I agree that GTK's C API is pretty nasty, so that's why I suggested using gtkmm. That way you get proper encapsulation and don't have to remember to free your ``g_object``s etc. I feel like GTK (or QT or something) is probably easier for making a proper GUI than imgui and I think OpenCV windows aren't very flexible.